src/Controller/MenuController.java

```diff
@@ -364,7 +364,7 @@ public static boolean autoCompleteOptionsControl(final String command, final Dia
                 editClassSubMenu(false, currentClass, diagram);
                 break;
             case("edit-relationships"):
-                editRelationshipsControl(shouldTerminate, diagram);
+                editRelationshipsControl(false, diagram);
```

Comment:
If false is going to be passed in here everytime, is there any reason the boolean parameter needs to exist?